src/Evolution/Systems/NewtonianEuler/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ set(LIBRARY_SOURCES
 
 add_spectre_library(${LIBRARY} ${LIBRARY_SOURCES})
 
-add_subdirectory(NewtonianEulerSources)
+add_subdirectory(Sources)
```

Comment:
This commit should be squashed with the previous one so each commit alone compiles

Reply:
oh yeah, I know. I just wanted to show the changes in the most explicit way :)